File: ITCProject/app/src/main/java/com/latihan/itcproject/MainActivity.java
package com.latihan.itcproject;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    //TODO 12. BUAT CODINGANNYA UNTUK DITAMPILIN

    private RecyclerView rvMinuman;
    ArrayList<Minuman> list = new ArrayList<>();
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        rvMinuman = findViewById(R.id.rv_item);

        list.addAll(MinumanData.getistData());
        showRecycler();
    }

    private void showRecycler() {
        rvMinuman.setLayoutManager(new LinearLayoutManager(this));
        MinumanAdapter minumanAdapter = new MinumanAdapter(list);
        rvMinuman.setAdapter(minumanAdapter);
    }
}

File: DataDummyK/app/src/main/java/com/latihan/datadummyk/MainActivity.kt
package com.latihan.datadummyk

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.recyclerview.widget.LinearLayoutManager
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    private val listDataName = ArrayList<Name>()

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        rv_heroes.setHasFixedSize(true)

        listDataName.addAll(getListDataName())
        showRecyclerList()
    }

    private fun getListDataName(): ArrayList<Name>{
        val newName = resources.getStringArray(R.array.data_nama)

        val listName = ArrayList<Name>()
        for (position in newName.indices){
            val newListName = Name(
                newName[position]
            )
            listName.add(newListName)
        }
        return listName
    }

    private fun showRecyclerList(){
        rv_heroes.layoutManager = LinearLayoutManager(this)
        val listNameAdapter = ListHeroAdapter(listDataName)
        rv_heroes.adapter = listNameAdapter
    }
}
